docker/php-fpm/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM phpdockerio/php56-fpm:latest
+WORKDIR "/application"
+
+# Fix debconf warnings upon build
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install selected extensions and other stuff
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install  php5-mysql php5-redis php5-sqlite php5-gd \
+    && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
+
+# Install git
+RUN apt-get update \
+    && apt-get -y install git \
+    && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
+
+RUN composer install
```

Comment:
Doesn't work for me, says that Composer couldn't find the commposer.json file in /application.